taxcalc/utils.py

```diff
@@ -357,7 +358,10 @@ def add_columns(pdf):
         columns = STATS_COLUMNS + [income_measure]
     else:
         columns = None
-    res = results(obj, cols=columns)
+    if isinstance(obj, pd.DataFrame):
+        res = results(obj, cols=columns)
+    else:
+        res = results(obj.records, cols=columns)
```

Comment:
@martinholmer do you think you should add an assertion statement: `assert isinstance(obj, Calculator)`?

Reply:
Yes, I tried that but we have never imported Calculator or Records into the utils.py file.

Reply:
Ok, I see.